package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "nypl-discovery",
```

Comment:
Should this one be changed? Seems like this is the app name rather than the github account

Reply:
Good point. Will live that as is.